raiden/network/pathfinding.py

```diff
@@ -183,6 +185,28 @@ def get_pfs_iou(
         raise ServiceRequestFailed(str(e))
 
 
+def request_last_iou(
+        url: str,
+        token_network_address: typing.Union[typing.TokenNetworkAddress, typing.TokenNetworkID],
+        sender: typing.Address,
+        receiver: bytes,
```

Comment:
Shouldn't this be `Address`?

Reply:
Probably. I was not sure where we use binary vs hex here, but `str` and `typing.Address` would make more sense than `typing.Address` and `bytes`.